packages/ozone/src/daemon/index.ts

```diff
@@ -0,0 +1,71 @@
+import AtpAgent from '@atproto/api'
+import Database from '../db'
+import { DaemonConfig } from './config'
+import DaemonContext from './context'
+import { EventPusher } from './event-pusher'
+import { EventReverser } from './event-reverser'
+import { ModerationService } from '../mod-service'
+import { Keypair } from '@atproto/crypto'
+import { createServiceAuthHeaders } from '@atproto/xrpc-server'
+
+export { EventPusher } from './event-pusher'
+export { EventReverser } from './event-reverser'
+
+export class OzoneDaemon {
```

Comment:
Daemon can run in-process with the application server (if you only have one app server) or as a separate daemon